RELEASE.md

```diff
@@ -44,6 +49,27 @@ For more info, see the release page: https://github.com/kubernetes-sigs/kubebuil
 
 2. An announcement email is sent to `kubebuilder@googlegroups.com` with the subject `[ANNOUNCE] Kubebuilder $VERSION is released`
 
+## What to do if things goes wrong? How to release from my local env as a workaround?
+
+As a workaround we can release from locally by:
+
+1. Download google container builder: https://github.com/GoogleCloudPlatform/cloud-build-local
+2. Verify that you can use the cloud-build-local CLI tool
+3. Ensure that you are locally in the branch created for the release
```

Comment:
We should specify the branch format here just to be extra verbose. Something like:
```suggestion
3. Ensure that you are locally in the branch created for the release (`release-<MAJOR.MINOR>`)
```